ui/page/send/page.go

```diff
@@ -381,7 +421,7 @@ func (pg *Page) constructTx() {
 		amountAtom = spendableAmount - feeAtom
 	}
 
-	wal := pg.WL.SelectedWallet.Wallet
+	wal := pg.selectedWallet
```

Comment:
My previous suggestion to change Load to Theme is because of this. Because there is a `pg.Load` property, different parts of this page could be accessing `pg.WL.SelectedWallet.Wallet` instead of `pg.selectedWallet`. You've probably corrected all such cases, but to be safe, let's take off Load completely.

Reply:
The have been updated to use `pg.selectedWallet.` from my previous  commit.